db/seeds.rb

```diff
@@ -8,36 +8,91 @@
 
 print 'Seeding Users...'
 
-User.create! [
-  {
-    email: 'tybot204@gmail.com',
+users = []
+unless User.exists? discord_name: 'Agthor#8442'
+  users << {
+    email:        'tybot204@gmail.com',
     discord_name: 'Agthor#8442',
-    password: 'password',
-    admin: true
-  }, {
-    email: 'justinrlaforge@gmail.com',
+    password:     'password',
+    admin:        true
+  }
+end
+unless User.exists? discord_name: 'iDreamPixels#3186'
+  users << {
+    email:        'justinrlaforge@gmail.com',
     discord_name: 'iDreamPixels#3186',
-    password: 'password',
-    admin: true
-  }, {
-    discord_name: 'Best 2hu#0550',
-    password: Faker::Internet.password(6, 72)
-  }, {
-    discord_name: 'Jibbers#2487',
-    password: Faker::Internet.password(6, 72)
-  }, {
-    discord_name: 'NicksPatties#4392',
-    password: Faker::Internet.password(6, 72)
-  }, {
-    discord_name: 'rebble#3220',
-    password: Faker::Internet.password(6, 72)
-  }, {
-    discord_name: 'RescuePenguin#8507',
-    password: Faker::Internet.password(6, 72)
-  }, {
-    discord_name: 'Sloth#6910',
-    password: Faker::Internet.password(6, 72)
+    password:     'password',
+    admin:        true
   }
-]
+end
+users << { discord_name: 'Best 2hu#0550' }      unless User.exists? discord_name: 'Best 2hu#0550'
+users << { discord_name: 'Jibbers#2487' }       unless User.exists? discord_name: 'Jibbers#2487'
+users << { discord_name: 'NicksPatties#4392' }  unless User.exists? discord_name: 'NicksPatties#4392'
+users << { discord_name: 'rebble#3220' }        unless User.exists? discord_name: 'rebble#3220'
+users << { discord_name: 'RescuePenguin#8507' } unless User.exists? discord_name: 'RescuePenguin#8507'
+users << { discord_name: 'Sloth#6910' }         unless User.exists? discord_name: 'Sloth#6910'
+User.create! users
+
+puts 'done'
+
+print 'Seeding Teams...'
+
+teams = []
+teams << { name: 'Alpha Team' } unless Team.exists? name: 'Alpha Team'
+teams << { name: 'Beta Team' }  unless Team.exists? name: 'Beta Team'
+Team.create! teams
+
+puts 'done'
+
+print 'Seeding Record Books...'
+
+record_books = []
+unless RecordBook.exists? name: 'Season 2'
+  record_books << {
+    name:            'Season 1',
+    published:       true,
+    start_time:      2.weeks.ago,
+    end_time:        2.weeks.from_now,
+    rush_start_time: 1.week.from_now,
+    rush_end_time:   2.weeks.from_now
```

Comment:
What's the difference between `start_time` and `rush_start_time`?

Reply:
This is down to how we've planned record books working. Each month we have a new record book, where only clan members can participate. The last week or so new applicants get to join in with a set of new challenges. That period is called "Rush Week" and has it's own time frame.